### PR TITLE
Add conditional schemas according to resource type

### PR DESCRIPTION
**Pull request by wfulgenzi**
* Rewording to make the RulesBuilder process simpler.
* Add conditional schemas to avoid confusing additional options that aren't appropriate for the given resource: E.g., JSONPaths property on a HTML Resource.

### CODE REVIEW

**Review by nico2che — APPROVED** [review]
📖

File: technology-rules/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@weglot/technology-rules",
-  "version": "3.24.1",
+  "version": "3.24.2",
```
Comment (roptch) [inline technology-rules/package.json:3]: C'est plus bon ça :willstle:

File: technology-rules/schemas/translations.schema.json [hunk 2]
```diff
@@ -16,139 +16,369 @@
...
+      "title": "JSON Paths",
+      "type": "array",
+      "uniqueItems": true,
+      "default": [],
+      "items": {
+        "type": "string"
+      }
+    },
+    "DefinitionXMLPathArray": {
+      "description": "A list of XML path (xpath) strings to identify the element(s) for translation. For more info see: w3schools.com/xml/xpath_syntax.asp",
```
Comment (roptch) [inline technology-rules/schemas/translations.schema.json:251]: C'est pas du xpath les sélecteurs pour les XML il me semble ?

**Comment by wfulgenzi** [comment]
This is no longer required since the Rules Builder is no longer integrated with the Technology Rules schema. We may come back to it if we want the schema to apply a validation on the rules.

**Pull request closed without merging**